addons/laser/stringtable.xml

```diff
@@ -6,18 +6,21 @@
             <Polish>Kod lasera</Polish>
             <French>Code laser</French>
             <Russian>Лазерный код</Russian>
+            <Spanish>Código láser</Spanish>

```

Comment:
Yeah, could be. Its about changing the frequency the laser works, so you can difference between them when various objectives are marked for CAS. I tried to find some references about that in Spanish but couldn't. But yeah, "Código del láser" sounds better.